lib/mixin/sentry_state_transaction_mixin.dart

```diff
@@ -0,0 +1,21 @@
+library after_layout;
+
+import 'package:flutter/widgets.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+mixin SentryStateTransactionMixin<T extends StatefulWidget> on State<T> {
+  ISentrySpan? transaction;
+
+  @override
+  void initState() {
+    super.initState();
+    transaction = Sentry.startTransaction(
+      runtimeType.toString(),
+      'ui.load',
+      bindToScope: true,
+    );
+    WidgetsBinding.instance?.addPostFrameCallback((_) {
+      transaction?.finish(status: SpanStatus.ok());
```

Comment:
See how we do to set status here https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java#L218-L223